arctic/_util.py

```diff
@@ -23,7 +23,7 @@ def are_equals(o1, o2, **kwargs):
         return False
 
 
-def enable_sharding(arctic, library_name, hashed=True):
+def enable_sharding(arctic, library_name, hashed=True, key='symbol'):
```

Comment:
I will add a docstring, 'symbol' is the only key that was used until now, but for BSONStore we would like other options.